### PR TITLE
V2.0.0

### PR DESCRIPTION
**Pull request by mac89**
Created v2.0.0 of the widget.

### CODE REVIEW

File: src/filterable.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                    };
+
+                    // Wait until one of the lazyloader events is triggered
+                    $element.one( lazyLoaderEvents, triggerFilter );
+
+                    // Set the searchQuery option to the new value
+                    widget.option( searchQueryOption, val );
+                }
+            } else {
+
```
Comment (rcolpaert) [inline src/filterable.js]: Remove empty line

File: src/lazyloader.js [hunk 2]
```diff
@@ -0,0 +1,348 @@
+"use strict";
+
+$.widget( "mobile." + widgetName, $.mobile.listview, {
```
Comment (rcolpaert) [inline src/lazyloader.js:3]: Missing type declarations for most options and variables

File: src/lazyloader.js [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+        threshold: 100,
+
+        /**
+         * The ajax request type.
+         */
+        ajaxType: "POST"
+    },
+
+    /**
+     * The varructor.
```
Comment (rcolpaert) [inline src/lazyloader.js]: What is a 'varructor'?

File: src/lazyloader.js (hunk 2 above)
Comment (rcolpaert) [reply src/lazyloader.js:3]: You could remove various unnecessary line breaks

**Review by rcolpaert — CHANGES_REQUESTED** [review]
Overall OK, just some minor JSDoc related remarks

File: src/filterable.js (hunk 1 above)
Comment (mac89) [reply src/filterable.js]: Linter requires an empty line between comment and the code line above.

**Pull request merged**